odk1-src/collect-offline-maps.rst

```diff
@@ -1,44 +1,68 @@
 .. spelling:: 
   basemap
+  Basemap
+  basemaps
+  Basemaps
   geospatial
   heatmaps
   Mapbox
   pbf
+  tileset
+  tilesets
 
 Using Offline Maps
 ====================
 
-Collect's :ref:`location-widgets` can be configured to display different maps. To use online maps, :ref:`set the mapping engine and basemap <mapping-settings>` in :ref:`interface-settings`. Users will need to be online to load questions with maps.
+Collect's :ref:`geopoint <geopoint-maps>`, :ref:`geotrace <geotrace-widget>`, and :ref:`geoshape <geoshape-widget>` questions can be configured to display different map data. To use online basemaps, :ref:`set the basemap source and style <basemap-settings>` in :guilabel:`mapping-settings`. Users will need to be online to load questions with maps.
```

Comment:
The last sentence might confuse people a bit?  Maybe identify both kinds of layers first, then separately say one type is online and one type is offline?  A rough attempt:

    ...different map data. The `Maps` page in `General Settings`
    lets you select a `Basemap` to show, as well as a `Reference
    layer` to show on top of the basemap.

    The data for all the available basemaps comes from services
    on the Internet, so the basemap will only be visible to users
    who are online.  To choose a basemap, select a `Source` and
    then a `Style`.

    For the reference layer, however, you can select a file on the
    device, and it will be visible offline.  Offline layers are useful
    for...